### ISSUE TITLE
Model predicts irrelevant images as correct prediction

### ISSUE THREAD

**Issue by thangaraj535** [alankbi/detecto#103]
![WhatsApp Image 2022-01-09 at 8 26 06 PM](https://user-images.githubusercontent.com/74810897/148922908-c1af31d6-b578-4266-9891-c9befb2ec677.jpeg)
![WhatsApp Image 2022-01-09 at 8 23 11 PM](https://user-images.githubusercontent.com/74810897/148922914-060943f5-1925-4cf3-8bce-32cd41b11dc7.jpeg)
![WhatsApp Image 2022-01-09 at 8 22 24 PM](https://user-images.githubusercontent.com/74810897/148922916-8a4719ee-69b8-4bbf-825c-cc43642419b6.jpeg)
![WhatsApp Image 2022-01-06 at 10 06 54 PM](https://user-images.githubusercontent.com/74810897/148922919-43df9dda-6371-4b14-9850-0d98944d51e3.jpeg)
![WhatsApp Image 2022-01-06 at 10 04 58 PM](https://user-images.githubusercontent.com/74810897/148922922-8784bd58-ce4c-4128-a680-2523039e0a27.jpeg)
![WhatsApp Image 2022-01-09 at 8 28 30 PM](https://user-images.githubusercontent.com/74810897/148922924-6945592d-9737-45bf-96d9-10d499b16afd.jpeg)
![WhatsApp Image 2022-01-09 at 8 27 12 PM](https://user-images.githubusercontent.com/74810897/148922927-544bc500-1049-45a4-b0a9-75c365793160.jpeg)

I have trained a custom model to predict diseases on crops leaf, while testing the model after training it is predicting a car image as disease spot with the accuracy of 99%.
Kindly provide the solution for this as soon as possible. I have also attached the actual predicted image

![download](https://user-images.githubusercontent.com/74810897/148923901-5a03e1ca-32f6-4526-af24-8dee4fdd9136.png)
![download (1)](https://user-images.githubusercontent.com/74810897/148923905-8dc3518d-4346-4a33-9a8b-aab6bcc89c65.png)

**Comment by alankbi** [comment]
It looks like your model might be overfitted or basically just producing too many false positives. What does your training dataset look like? It might be worth going back and trying to make sure you have a wide and varied range of images to train on - I think it'd also help if several of the target objects are smaller/don't take up the whole image, that way the model doesn't learn to just predict the entire image to be a bacterial spot every time.
